Directory.Packages.props

```diff
@@ -10,6 +10,8 @@
 
     <LombiqHelpfulLibrariesVersion>12.1.0</LombiqHelpfulLibrariesVersion>
     <LombiqTestsUIVersion>12.1.1-alpha.14.osoe-1057</LombiqTestsUIVersion>
+    <!-- This needs to be upgraded to 9.0. It's currently fixed to 8.0 for CI to build without issues.-->
+    <AnalysisLevel>8.0</AnalysisLevel>
```

Comment:
Fixed AnalysisLevel to 8.0 but I may try upgrading it now that I found the issue with the CI build.